nifi-extension-bundles/nifi-poi-bundle/nifi-poi-services/src/test/java/org/apache/nifi/excel/TestExcelSchemaInference.java

```diff
@@ -50,69 +45,54 @@ public class TestExcelSchemaInference {
+
     private final TimeValueInference timestampInference = new TimeValueInference("MM/dd/yyyy", "HH:mm:ss.SSS", "yyyy/MM/dd/ HH:mm");
 
     @Mock
     ComponentLog logger;
 
-    @ParameterizedTest
-    @MethodSource("getLocales")
-    public void testInferenceAgainstDifferentLocales(Locale locale) throws IOException {
-        final Map<PropertyDescriptor, String> properties = new HashMap<>();
-        new ExcelReader().getSupportedPropertyDescriptors().forEach(prop -> properties.put(prop, prop.getDefaultValue()));
-        final PropertyContext context = new MockConfigurationContext(properties, null, null);
-
-        try (final InputStream inputStream = getResourceStream("/excel/numbers.xlsx")) {
-            final InferSchemaAccessStrategy<?> accessStrategy = new InferSchemaAccessStrategy<>(
-                    (variables, content) -> new ExcelRecordSource(content, context, variables, logger),
-                    new ExcelSchemaInference(timestampInference, locale), logger);
-            final RecordSchema schema = accessStrategy.getSchema(null, inputStream, null);
-            final List<String> fieldNames = schema.getFieldNames();
-            assertEquals(Collections.singletonList(EXPECTED_FIRST_FIELD_NAME), fieldNames);
-
-            if (Locale.FRENCH.equals(locale)) {
-                assertEquals(RecordFieldType.STRING, schema.getDataType(EXPECTED_FIRST_FIELD_NAME).get().getFieldType());
-            } else {
-                assertEquals(RecordFieldType.CHOICE.getChoiceDataType(RecordFieldType.FLOAT.getDataType(), RecordFieldType.STRING.getDataType()),
-                        schema.getDataType(EXPECTED_FIRST_FIELD_NAME).get());
-            }
-        }
-    }
-
-    private static Stream<Arguments> getLocales() {
-        Locale hindi = Locale.of("hin");
-        return Stream.of(
-                Arguments.of(Locale.ENGLISH),
-                Arguments.of(hindi),
-                Arguments.of(Locale.JAPANESE),
-                Arguments.of(Locale.FRENCH)
-        );
-    }
-
     @Test
     public void testInferenceIncludesAllRecords() throws IOException {
         final Map<PropertyDescriptor, String> properties = new HashMap<>();
         new ExcelReader().getSupportedPropertyDescriptors().forEach(prop -> properties.put(prop, prop.getDefaultValue()));
         final PropertyContext context = new MockConfigurationContext(properties, null, null);
 
         final RecordSchema schema;
-        try (final InputStream inputStream = getResourceStream("/excel/simpleDataFormatting.xlsx")) {
+        try (final InputStream inputStream = getResourceStream()) {
             final InferSchemaAccessStrategy<?> accessStrategy = new InferSchemaAccessStrategy<>(
                     (variables, content) -> new ExcelRecordSource(content, context, variables, logger),
                     new ExcelSchemaInference(timestampInference), Mockito.mock(ComponentLog.class));
```

Comment:
No need to call `Mockito.mock(ComponentLog.class)` as we already have an instance of ComponentLog annotated with the `@Mock` annotation.
```suggestion
                    new ExcelSchemaInference(timestampInference), logger);
```